Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -415,6 +417,43 @@ def get_remediation_details():
         demisto.results('No Remediation Details Found')
 
 
+def redlock_search_config():
+    """
+    Run query in config
+    """
+    query = demisto.args().get('query', None)
+    if not query:
+        return_error('You must specify a query to retrieve assets')
+    payload = {
+        'query': query,
+        'limit': 100,
+        'sort': [{"direction": "desc", "field": "insertTs"}],
+        'withResourceJson': True
+    }
+    handle_time_filter(payload, {'type': 'to_now', 'value': 'epoch'})
```

Comment:
should be a command argument

Reply:
@yaakovi not exactly sure what you mean here, the command uses the filters provided as imputs